utilities.py

```diff
@@ -79,6 +82,9 @@ def parse_config(config_filename):
         else:
             log_file = ''
 
+        petrarch_version = parser.get('Petrarch', 'petrarch_version')
+        print("petrarch version is {}".format(petrarch_version))
```

Comment:
Probably don't want to put another print statement in here. PETR and the pipeline already print out a _lot_ of information.